content/scout/policy/_index.md

```diff
@@ -52,12 +52,13 @@ open-source licenses.
 
 ## Default policies
 
-Docker Scout ships the following three out-of-the-box policies:
+Docker Scout ships the following out-of-the-box policies:
 
 - [Critical and high vulnerabilities with fixes](#critical-and-high-vulnerabilities-with-fixes)
 - [Critical vulnerabilities](#critical-vulnerabilities)
 - [Packages with AGPLv3, GPLv3 licenses](#packages-with-agplv3-gplv3-licenses)
 - [Base images not up-to-date](#base-images-not-up-to-date)
+- [High-profile CVEs](#high-profile-cves)
```

Comment:
Let's change CVEs to vulnerabilities to keep the name consistent with other policies. I've asked @felipecruz91 to make the necessary code change as well.